ajax/libs/angular-tree-control/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "angular-tree-control",
+  "filename": "angular-tree-control.min.js",
+  "version": "0.2.25",
+  "description": "Angular Tree Control",
+  "homepage": "https://wix.github.io/angular-tree-control",
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/wix/angular-tree-control.git"
+  },
+  "keywords": [
+    "angular-tree-control",
+    "Angular JS Tree",
+    "tree control component"
+  ],
+  "author": "Yoav Abrahami",
+  "license": "ISC",
```

Comment:
@pvnr0082t 
I see the "MIT" license in [LICENSE](https://github.com/wix/angular-tree-control/blob/master/LICENSE)
Maybe you should confirm it.
Thanks.